tests/cdrouter_bootstrap.py

```diff
@@ -58,6 +58,16 @@ def runTest(self):
 
         c = CDRouter(self.cdrouter_server)
 
+        """Go to cdrouter frequency and reboot to Dual-stack config"""
+        wan_ip = board.get_interface_ipaddr("wan0")
+        if self.config.board['cdrouter_location'] == 'CBN':
```

Comment:
I don't think using the location is the right thing to flag off of here.

Reply:
This location is used for testing cdrouter with another way. The NTL team(maual test team) using cdrouter to go to specific frequency to test case, not we usually using frequency. Therefore, I just using this location to change frequency. Is it proper?